### ISSUE TITLE
EPUB A11y Display Guidelines Hazards no_sound_hazard Variable Definition Missing from Section 3.5.2

### ISSUE THREAD

**Issue by jgaydos-OD** [w3c/publ-a11y#717]
In [Section 3.5.2](https://w3c.github.io/publ-a11y/a11y-meta-display-guide/2.0/draft/techniques/epub-metadata/index.html#variables-setup-5), the variable `no_sound_hazard` is not defined. It is defined in Section 3.5.1 and used in Section 3.5.3.

I assume that it should be gathered via this XPath: `/package/metadata/meta[@property="schema:accessibilityHazard" and normalize-space()="noSoundHazard"]`

**Comment by clapierre** [comment]
Good catch, I will create a PR to fix this issue.
